CONTRIBUTING.md

```diff
@@ -90,9 +90,9 @@ RUST_LOG=DEBUG cargo test --package containerd-shim-wasmtime --lib -- wasmtime_t
 
 ### End to End tests
 
-The e2e test run on [k3s](https://k3s.io/) and [kind](https://kind.sigs.k8s.io/).  A test image is built using [oci-tar-builder](./crates/oci-tar-builder/) and is loaded onto the clusters.  This test image is not pushed to an external registry so be sure to use the Makefile targets to build the image and load it on the cluster.
+The e2e test run on [k3s](https://k3s.io/) and [kind](https://kind.sigs.k8s.io/).  A test image is built using [oci-tar-builder](https://github.com/containerd/runwasi/tree/main/crates/oci-tar-builder) and is loaded onto the clusters.  This test image is not pushed to an external registry so be sure to use the Makefile targets to build the image and load it on the cluster.
```

Comment:
I think floating links like this would be most desirable in the docs since a PR moving or removing the file should cause a failure of the link check.

Reply:
what are floating links? Do you mean links like `./crates/oci-tar-builder/`? The issue with this is that in `runwasi.dev` it can't read links outside of the `/docs` folder so the linkChecker would fail.